src/main/java/com/epam/DAO/PersonService.java

```diff
@@ -0,0 +1,33 @@
+package com.epam.DAO;
+
+import com.epam.model.Person;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate;
+
+
+import java.sql.ResultSet;
+import java.util.List;
+
+@Service
+public class PersonService {
```

Comment:
I think you should make sure that there is the final version of your code here. 'Cause I can find PersonService + PersonDAO